sections/kubecontext.zsh

```diff
@@ -36,8 +36,27 @@ spaceship_kubecontext() {
     [[ -n $kube_namespace && "$kube_namespace" != "default" ]] && kube_context="$kube_context ($kube_namespace)"
   fi
 
+  local section_color
+
+  # Apply custom color to section if $kube_context matches a pattern defined in SPACESHIP_KUBECONTEXT_COLORGROUPS array.
+  # See Options.md for usage example.
+  local len=${#SPACESHIP_KUBECONTEXT_COLORGROUPS[@]}
+  if [[ $len -gt 0 ]]; then
```

Comment:
I'd remove this `if`, we won't enter `for` loop below if the array is empty anyway. Less code is better 🙂

Reply:
valid point :)